sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM golang:1.12 as base-build
```

Comment:
It looks like this "base-build" image is only used for building Firecracker and not any Go programs.  Can you use the `rust:1.32` image instead and skip the need to install with rustup?